woocommerce/facebook-for-woocommerce
# Updates readme.txt to meet WordPress requirements

### Changes proposed in this Pull Request:



Closes #2695.


This PR updates the `readme.txt file` to meet the WordPress requirements.

* Keeps only the recent version changelog in the readme.txt file
* Updates the tags
* Updates the short description

### Detailed test instructions:
* Checkout the readme.txt file from this PR
* Ensure the WordPress requirements are met as described in #2695.


### Changelog entry

> Tweak - Updates readme.txt to meet WordPress requirements
